### PR TITLE
Improved Ukrainian translations

### PR DESCRIPTION
**Pull request by JackRainy**
* Translations for the Japan nation
* Improved translation of policies
* Spelling mistakes are corrected, etc.

### CODE REVIEW

File: android/assets/jsons/translationsByLanguage/Ukrainian.properties [hunk 1]
```diff
@@ -668,10 +668,10 @@ An enemy [RangedUnit] has destroyed the defence of [cityname] = Ворожий 
...
-Our [interceptorName] intercepted and destroyed an enemy [attackerName] = Наш підрозділ [interceptorName] перехопив і знищив ворожий підрозділ [attackerName]
-Our [$attackerName] was attacked by an intercepting [$interceptorName] = Наш підрозділ [$attackerName] атаковано ворожим перехоплювачем [$interceptorName]
-Our [$interceptorName] intercepted and attacked an enemy [$attackerName] = Наш підрозділ [$interceptorName] перехопив і атакував ворожий підрозділ [$attackerName]
+Our [attackerName] was destroyed by an intercepting [interceptorName] = Наш [attackerName] знищено при перехопленні ворожим підрозділом [interceptorName]
+Our [interceptorName] intercepted and destroyed an enemy [attackerName] = Наш [interceptorName] перехопив і знищив ворожий [attackerName]
+Our [$attackerName] was attacked by an intercepting [$interceptorName] = Наш [$attackerName] атаковано ворожим перехоплювачем [$interceptorName]
+Our [$interceptorName] intercepted and attacked an enemy [$attackerName] = Наш [$interceptorName] перехопив і атакував ворожий [$attackerName]
```
Comment (Oheto) [inline android/assets/jsons/translationsByLanguage/Ukrainian.properties]: this translation is incorrects with neutral and feminine gender

Comment (JackRainy) [reply android/assets/jsons/translationsByLanguage/Ukrainian.properties]: Taking into account that all interceptors and bombers are male units, it is acceptable to simplify that.
Беручи до уваги, що усі повітряні юніти чоловічого роду, то переклад тут можна спростити.

File: android/assets/jsons/Nations/Nations_Ukrainian.json [hunk 2]
```diff
@@ -377,28 +377,28 @@
...
+		neutralNo:["Вибач.","Ні.","Мабуть, наступного разу."]
+		neutralYes:["Дуже добре.","Згоден.","Хай. (Так)"]
 
-		hateHello:"Oh, it's you..."
-		hateLetsHearIt:["Let me hear it.","Go on","Proceed"]
-		hateNo:["Ie. (No)","Excuse me?","Maybe another time"]
-		hateYes:["I agree.","Oh... very well.","There is probably no other way."]
+		hateHello:"Ох, це ти..."
+		hateLetsHearIt:["Давай я послухаю.","Давай далі.","Продовжуй."]
+		hateNo:["Ее. (Ні)","Перепрошую?","Мабуть, наступного разу."]
```
Comment (Oheto) [inline android/assets/jsons/Nations/Nations_Ukrainian.json]: `another time`, maybe, better translate as `іншого разу`

Comment (JackRainy) [reply android/assets/jsons/Nations/Nations_Ukrainian.json]: Maybe, you are right.
I will correct that in the next translations as `Мабуть, іншим разом.`

**Pull request merged**